common_design.libraries.yml

```diff
@@ -198,6 +198,11 @@ cd-table:
     - common_design/css-vars-ponyfill
     - common_design/array-from-polyfill
 
+cd-tag:
+  css:
+    component:
+      components/cd-tag/cd-tag.css: {}
+
```

Comment:
Because we are using CSS custom properties, each library needs the ponyfill set as a dependency so it loads in IE11.
See https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/common_design.libraries.yml#L70